Notes

JonWuIDM opened the pull request:
Established what I wanted to do, but how would I implement a
pushMatrix/popMatrix for the background every time a key is pressed.

JonWuIDM committed:
Notes

Established what I wanted to do, but how would I implement a
pushMatrix/popMatrix for the background every time a key is pressed.

LadyK commented:
Jonatan, remember that all the graphic code that must be pushed and poped, must be nested inside a push/pop. If you have more thoughts and comments in your code, I can make other suggestions. In the mean time, I'd like to see you incorporate interactivity, movement and even vectors and physics. Check out Daniel Shiffman's Nature of Code text + video's. Keep it coming!

JonWuIDM committed:
Notes

Remade the entire thing, WASD keys work but they don't actually move
anything.

LadyK commented:
Hi Jonatan. It's good to see the development, but it can go much further. You have control and passing parameters to functions down, which is good. But, I'd like to see more happening here with reference to graphics. How could you incorporate motion or different states to convey your adjective? What is your adjective? Could someone else get an idea of what your adjective is just by looking at this?

JonWuIDM committed:
Project_1

It rotates on the current 0,0 point, I wanted to make it rotate around
the shield's own center, which I can make the ellipse do but the star's
cords have to modified so much, along with the lines for Anarchy() and
flashbng()
